src/categories/topics.js

```diff
@@ -141,7 +142,11 @@ module.exports = function (Categories) {
 			});
 			return result && result.pinnedTids;
 		}
-		const pinnedTids = await db.getSortedSetRevRange(`cid:${data.cid}:tids:pinned`, data.start, data.stop);
+		let pinnedTids = await db.getSortedSetRevRange(`cid:${data.cid}:tids:pinned`, data.start, data.stop);
+		const canSchedule = await privileges.categories.can('topics:schedule', data.cid, data.uid);
```

Comment:
This is very minor, but you can run these two with `Promise.all` since they don't depend on each other.

```
const [allPinnedTids, canSchedule ] = await Promise.all([
  db.getSortedSetRevRange(`cid:${data.cid}:tids:pinned`, data.start, data.stop),
  privileges.categories.can('topics:schedule', data.cid, data.uid),
]);
```

Reply:
Done. :+1: